app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,132 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
 import { json } from '@remix-run/node';
 
+import { Button } from '~/components/ui/button';
+
+import { Image } from '~/components/ui/images';
+
 export const loader: LoaderFunction = async ({ params }) => {
```

Comment:
Future work: Replace `LoaderFunction` with `LoaderArgs` so TS can infer the data correctly.

Reply:
Even though every name sounds semantically correct, it should be LoaderFunctionArgs 🙂